enginetest/queries/script_queries.go

```diff
@@ -3918,6 +3918,23 @@ var CreateCheckConstraintsScripts = []ScriptTest{
 			},
 		},
 	},
+	{
```

Comment:
`create table ... like` tests probably make more sense to live with the other `create table` tests. Check out `TestCreateTable` and `queries.CreateTableQueries` and see if you can find a nice spot to nestle this into.

Reply:
Okay. I moved the test into create_table_queries.go. How's this?

Reply:
Looks great! We have soooo many tests, it's helpful to try and keep them as organized as we can so they don't sprawl too much. 

Ship it! 🚢